connectors/bigtable/flink-connector-gcp-bigtable/pom.xml

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.google.flink.connector.gcp.bigtable</groupId>
+    <artifactId>flink-connector-bigtable-parent</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
can we pull this version from the parent pom? same with some of the other dependencies (ie, mockito -- which maybe we dont need to define twice)

Reply:
Could we work on implementing the POM with parent separately? I think we should talk about if we want to build everything from root or not.  

If we add multiple connectors, we may want to allow separate more easily. I am aware we can do `mvn package -pl CONNETOR_MODULE`.

Reply:
yeah that's fine